### PR TITLE
Organization-wide Workflow Add Instructions

### PR DESCRIPTION
**Pull request by rohankh532**
Updated the README with instructions on using [multi-repo-action](https://github.com/ossf/scorecard-action/tree/main/multi-repo-action) to add the workflow to every repository under an organization.

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@azeemshaikh38 @olivekl  This has been open for a while. Can we get some 👀 ? So that we can decide to merge or add some more fixes to this.

File: README.md [hunk 1]
```diff
@@ -75,6 +75,17 @@ Then click "Add More Scanning Tools."
 
 ![image](/images/install04.png)
 
+### Workflow Setup - Organization Wide
+The scorecard workflow can be automatically added to every repository under an organization using the [multi-repo-action](https://github.com/ossf/scorecard-action/tree/main/multi-repo-action) tool.
```
Comment (olivekl) [inline README.md:79]: Suggested rewording (minor):

`To add the Scorecard workflow to every repository under an organization, use the multi-repo-action tool. Follow these steps:

File: README.md [hunk 2]
```diff
@@ -75,6 +75,17 @@ Then click "Add More Scanning Tools."
...
+1) `git clone ` this repository & `cd multi-repo-action`
```
Comment (olivekl) [inline README.md:80]: Specify which repository you mean by "this repository" -- i.e., `git clone the cd multi-repo-action repository and this repository (scorecard-action)`

**Review by olivekl — APPROVED** [review]
Minor comments, lgtm otherwise

**Comment by naveensrinivasan** [comment]
> Minor comments, lgtm otherwise

I am going to take over this PR and address those comments. Thanks@rohankh532!

**Comment by naveensrinivasan** [comment]
Closing this for #773

**Pull request closed without merging**